src/vs/workbench/contrib/files/browser/fileActions.ts

```diff
@@ -710,6 +711,16 @@ export function validateFileName(item: ExplorerItem, name: string): { content: s
 	return null;
 }
 
+function escapeMarkdown(filename: string): string {
+	if (!filename) {
```

Comment:
No need for this check we are using non nullable types

Reply:
without the check, it didnt pass [CI](https://github.com/microsoft/vscode/runs/1979231440)